client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-feature-previews/jetpack-preview-pane.tsx

```diff
@@ -43,14 +31,23 @@ export function JetpackPreviewPane( {
 
 	const { selectedSiteFeature, setSelectedSiteFeature } = useContext( SitesDashboardContext );
 
+	useEffect( () => {
+		if ( selectedSiteFeature === undefined ) {
+			setSelectedSiteFeature( 'jetpack_boost' );
+		}
+		return () => {
+			setSelectedSiteFeature( undefined );
+		};
+	}, [] );
```

Comment:
Here we control the default tab selected, in this case, `jetpack_boost`.

Also, it unset the selected feature when the component is dismounted.

Reply:
Here @gogdzl